src/.vuepress/theme/layouts/HomeLayout.vue

```diff
@@ -89,11 +89,11 @@
       ></div>
     </div>
 
-    <div class="flex flex-col pt-36 mb-10 z-[20]">
+    <div class="flex flex-col pt-36 z-[20]">
       <div class="flex flex-col mb-44">
         <h1 class="mb-8 text-3xl lg:text-5xl">{{ data.introductionTitle }}</h1>
         <p
-          class="w-[1040px] max-w-[90vw] text-center mt-O mb-12 mx-auto text-base lg:text-2xl"
```

Comment:
I put this breakpoint to better match small device width, we have a use case that requires removing it?

Reply:
I just thought it looked a bit better this way but we can switch it back. I dont think we have mobile designs for the whole page, we should also make sure the fonts are consistent across the different sections too.